language/ir-testsuite/tests/misc/ignore_checks_when_exp_file_present.exp

```diff
@@ -0,0 +1,3 @@
+[0], Transaction(0)
+[1], Move VM Status: EXECUTED
```

Comment:
Looks good! It would be cool to get the VM Error full dump in there some day, but not needed for this PR

Reply:
It already does that! I've added a transaction with an `ARITHMETIC_ERROR ` to the test and it prints out `[7], Move VM Status: EXECUTION_FAILURE { status_code: ARITHMETIC_ERROR, location: Script, function_definition: 0, code_offset: 3 }`. (Ignore the `,` after `[7]`. I'll get it removed before landing.)` Counterintuitively, `EXECUTE` is a complete status as there's nothing wrong to report. I had the same doubt when implementing this because it just looked too short to be complete.

Reply:
That is the VM Status though, Not the VMError, these are distinct things. The VM Error does not propagate past the Move VM (i.e. it is captured in the Libra VM/adaptor)

Reply:
Ah I see the VMError will be able to provided unfiltered and more granulated information. Seems like a problem to be solved by the Move-only testing flow I'm building now.

Reply:
It will also be helpful for testing the bytecode verifier, as you get the offsets for the errors (again probably a separate flow there)